components/service/sync-autofill/src/main/java/mozilla/components/service/sync/autofill/GeckoCreditCardsAddressesStorageDelegate.kt

```diff
@@ -8,8 +8,10 @@ import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.Deferred
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.async
+import kotlinx.coroutines.launch
```

Comment:
This is kinda of Gecko specific. It is only consumed by the GeckoAutocompleteStorageDelegate. Its respective Login counterpart also uses the Gecko prefix. If we want to rename this, I would suggest we do it for both, and note that it will be a breaking change.